sentences/ca/_common.yaml

```diff
@@ -1,12 +1,66 @@
 language: ca
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: Ho sento, no entenc això
+    no_area: No hi ha l'àrea {{ area }}
+    no_domain: No hi ha {{ domain }} a {{ area }}
+    no_device_class: No hi ha {{ device_class }} a {{ area }}
+    no_entity: No hi ha cap dispositiu o entitat que es digui {{ entity }}
+    handle_error: He trobat un error inesperat mentre feia l'intent
+lists:
+  color:
+    values:
+      - in: blanc
+        out: white
+      - in: negre
+        out: black
+      - in: vermell
+        out: red
+      - in: taronja
+        out: orange
+      - in: groc
+        out: yellow
+      - in: verd
+        out: green
+      - in: blau
+        out: blue
+      - in: "(lila|morat|porpre)"
```

Comment:
Lists cannot use matching syntax. Make an entry per value

Reply:
So I guess it _can_ use a matching syntax. @synesthesiam do we want that?

Reply:
Waiting for your input / suggestion on how to proceed. @synesthesiam

Reply:
For now, it's ok. We need to fix it in a few other places too if we decide to ban it.